APACHE_2.0

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think this file is optional (LICENSE has a link to it)

Reply:
I saw that it is usually rewritten there, also in cmssw.